packages/chakra-components/src/MenuBar/SubMenu.js

```diff
@@ -159,26 +136,13 @@ const SubMenu = ({
     resetTabIndex();
   };
 
-  const closeMenuWithoutIndex = () => {
-    if (!isControlled) {
-      setIsOpen(false);
-    }
-
-    if (onClose) {
-      onClose();
-    }
-
-    resetTabIndex();
-  };
-
   const context = {
     activeIndex,
     isOpen: _isOpen,
     focusAtIndex,
     focusOnLastItem,
     focusOnFirstItem,
     closeMenu,
-    closeMenuWithoutIndex,
```

Comment:
Why was this removed?

Reply:
Kindly add this as well. 
Go through and understand, what this function does and what doesn't happen removing this.
Let me know if you don't understand.